Resources/Prototypes/Catalog/VendingMachines/nutri.yml

```diff
@@ -3,3 +3,14 @@
   name: NutriMax
   description: A vending machine containing nutritional substances for plants and botanical tools.
   spriteName: nutri
+  startingInventory:
+    Spade: 3
+    PlantBGoneSpray: 20
+    WeedSpray: 20
+    PestSpray: 20
+    #syringe
```

Comment:
We already have syringes, the prototype ID is just `Syringe`

Reply:
Added thanks, wasn't sure if they were in and working.